### DISCUSSION TITLE
blog/how-to-resize-storage-in-aws-ec2

### DISCUSSION THREAD

**Discussion by giscus** [Mandy9943/techify#5] [Announcements]
# blog/how-to-resize-storage-in-aws-ec2

Learn how to quickly and safely resize your AWS EC2 instance storage when you are running out of disk space. This comprehensive guide provides both a quick solution and a detailed walkthrough with screenshots for Linux-based instances.

https://techify.blog/blog/how-to-resize-storage-in-aws-ec2

**Comment by Mandy9943** [comment]
hi
